frameworks/FSharp/frank/src/App/Program.fs

```diff
@@ -0,0 +1,126 @@
+module App.App
+
+open System
+open System.IO
+open System.Threading.Tasks
+open Microsoft.AspNetCore.Hosting
+open Microsoft.AspNetCore.Http
+open Dapper
+open Frank.Builder
+open FSharp.Control.Tasks
+open Npgsql
+open App
+open Models
+
+let private DefaultCapacity = 1386
+let private MaxBuilderSize = DefaultCapacity * 3
+let private BufferSize = 27
+
+type MemoryStreamCache = 
+    
+    [<ThreadStatic>]
+    [<DefaultValue>]
+    static val mutable private instance: MemoryStream
+
+    static member Get() = MemoryStreamCache.Get(DefaultCapacity)
+    static member Get(capacity:int) = 
+        
+        if capacity <= MaxBuilderSize then
+            let ms = MemoryStreamCache.instance;
+            let capacity = max capacity DefaultCapacity
+            
+            if ms <> null && capacity <= ms.Capacity then
+                MemoryStreamCache.instance <- null;
+                ms.SetLength 0L
+                ms
+            else
+                new MemoryStream(capacity)
+        else
+            new MemoryStream(capacity)
+
+    static member Release(ms:MemoryStream) = 
+        if ms.Capacity <= MaxBuilderSize then
+            MemoryStreamCache.instance <- ms
+
+let inline contentLength x = new Nullable<int64> ( int64 x )
+
+let json' data : HttpContext -> Task<unit> =
+    let bytes = Utf8Json.JsonSerializer.Serialize(data)
```

Comment:
@panesofglass Apologies for taking so long to review.

The object for the json test needs to be instantiated and serialized on every request.

`struct {| message = "Hello, World!" |}` and this serialization should be moved into the `ctx` function below.

Reply:
Also, I just looked at some other F# implementations and I see that Zebra has this problem since 2018

Reply:
Ah, thank you! I'll update.

Reply:
Hmm, this is actually correct. The object is created on each request and passed to the `json'` function where it is serialized. This is not cached anywhere ahead of time. This matches the F# Giraffe implementation, as well.